src/common/internal/query/compare.ts

```diff
@@ -58,7 +58,7 @@ function compareOneLevel(ordering: Ordering, aIsBig: boolean, bIsBig: boolean):
   return Ordering.Unordered;
 }
 
-function comparePaths(a: readonly string[], b: readonly string[]): Ordering {
+export function comparePaths(a: readonly string[], b: readonly string[]): Ordering {
```

Comment:
Since this is exported now, it should have documentation.
```suggestion
/** Compare two file paths, or file-local test paths, returning an Ordering between the two. */
export function comparePaths(a: readonly string[], b: readonly string[]): Ordering {
```

Reply:
Done